05_data_model_extensions.sql

```diff
@@ -20,6 +20,19 @@ added solely for QGEP';
 ALTER TABLE qgep_od.wastewater_structure ADD COLUMN _label text;
 COMMENT ON COLUMN qgep_od.wastewater_structure._label IS 'not part of the VSA-DSS data model
 added solely for QGEP';
+ALTER TABLE qgep_od.wastewater_structure ADD COLUMN _cover_label text;
+COMMENT ON COLUMN qgep_od.wastewater_structure._cover_label IS 'not part of the VSA-DSS data model
```

Comment:
Can you add some description of the content to this comment (what it's generated from and what is in there), the VSA-DSS parts are in my opinion mostly a comment for the comment ;)